actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -80,12 +78,22 @@ def format(formatter, filter = nil)
         formatter.result
       end
 
+      def self.defaults_filter_for(filter)
+        {
+            controller: /#{filter}/,
+            action: /#{filter}/
+        }
```

Comment:
What if we have namespaced `admin` routes? Are those still found? Let's add a test.